pkg/redis/redis_client.go

```diff
@@ -39,6 +39,7 @@ type Client interface {
 	HKeys(string) ([]string, error)
 	HGet(string, string) (string, error)
 	HSet(string, string, interface{}) error
+	HSetNX(key string, field string, value interface{}) (bool, error)
```

Comment:
the parameters are named here. Maybe also use unnamed parameters here for consistency?

Reply:
named parameters are wanted as of now, because a user of the interface should not be required to read the implementation